### DISCUSSION TITLE
o.T. (carré noir): Exercise 3

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#6] [Show and tell]
1.  Reopen file if needed: `exercises/03-ot/exercise.Rmd`
2.  Review the code chunk that contains the `make_exercise_molnar()` function which encompasses the whole system.
3.  Update the arguments `degree_pattern_options=` and `degree_sample_size=` for the function `make_exercise_molnar()` to experiment with modifying the rotation patterns to one of your choice. (Degrees options are 0, 90, 80, 270)
4.  Save your image using `ggsave()` code provided and share on GitHub discussion board.
5.  If you have extra time, here are some suggestion for customizing your piece:

-   Experiment with colors in the `fill=` and `color=` in `geom_polygon()`
-   Experiment with colors in the `plot.background` in `theme()`

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/14d41b4a-2bb2-40b0-8154-20c1d660649e)

**Comment by MHanken** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/49319823/c05e1d2b-5225-413e-ada5-6ed0e9d9aef8)

**Comment by sshaughnessy** [comment]
![v-rotatoin](https://github.com/posit-conf-2023/creative-coding/assets/36434962/b6312bc5-3063-47b0-93d8-b4837bb5e469)
![blue v](https://github.com/posit-conf-2023/creative-coding/assets/36434962/2238e42e-7d55-4741-a497-ddfe6a97384e)

**Comment by shaunmwilkinson** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362966/f55def36-34c9-4979-9eb8-eef9d4f1bcb9)

**Comment by roBertus31** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/25853405/1c77e3d5-7257-4564-8c95-65697760564a)

**Comment by Kim-s-h** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/22897443/b3b1da4e-cf65-4fbf-a560-3d37acd5c5b5)

**Comment by maxrogow** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/145362959/a71abb35-f0ae-4205-8803-e6d0a9d3ca1d)

**Comment by bshk93** [comment]
![ex3-piece-3](https://github.com/posit-conf-2023/creative-coding/assets/65432465/4a6dc998-8b2b-4d6b-9a02-1e59aafdd8b6)

**Comment by thomaspara** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/10082327/00847a77-fb8a-455d-a56a-726cfd68a942)

**Reply by thomaspara** [reply]
![ex3-piece2](https://github.com/posit-conf-2023/creative-coding/assets/10082327/93ebc406-f7fd-408c-9c43-871fc053d709)

**Comment by dougtommet** [comment]
![ex3-piece](https://github.com/posit-conf-2023/creative-coding/assets/7944225/a03c551c-b746-4c30-bb8d-16f431b7aaf4)

**Comment by lwdozal** [comment]
![ex3-piece_blue2](https://github.com/posit-conf-2023/creative-coding/assets/19821171/9f06d360-54e3-4bd3-bcfd-be5f3c07a8e5)
![ex3-pieceBlue](https://github.com/posit-conf-2023/creative-coding/assets/19821171/04e274fa-d28a-4ac2-8d78-bc8f605ae043)
![ex3-piece_pink](https://github.com/posit-conf-2023/creative-coding/assets/19821171/fd72690d-7a7e-4392-b328-6f26f99dcb5c)
![ex3-pieceSand](https://github.com/posit-conf-2023/creative-coding/assets/19821171/a774036a-1849-437f-8093-05b921a7cfc4)
![ex3-pieceL](https://github.com/posit-conf-2023/creative-coding/assets/19821171/cd486f8e-8380-472e-99e2-5b42929017bf)

**Comment by natalieoshea** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/10425075/9f3741c5-d5f3-4478-ab17-2b75dbd8008c)

**Comment by datatoolbox** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/5366795/92c1305f-ec21-4556-a596-a02f7af13991)
